internal/runtime/client/client.go

```diff
@@ -485,6 +496,7 @@ func httpCall(ctx context.Context, request, response runtime.Object, opts *httpC
 	}
 
 	if requireConversion {
+		log.V(5).Info(fmt.Sprintf("Hook version of received response is different. Converting response back to %s", opts.hookGVH))
```

Comment:
```suggestion
		log.V(5).Info(fmt.Sprintf("Supported hook version of extension is different. Converting response back to %s", opts.hookGVH))
```
for consistency with the log we have above

Reply:
TBH i like better this one with a small change (`Hook version of received response is %s.`)
What about converting both to this format?

Reply:
Fine for me. I would only like to keep them a bit consistent